<ui version="4.0" >
 <class>Information</class>
 <widget class="QDialog" name="Information" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>496</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>462</width>
    <height>496</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>462</width>
    <height>496</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Information</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/buttons/images/information.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frameHeader" >
     <property name="styleSheet" >
      <string>QFrame{
background: white;
} </string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="imgIcon" >
        <property name="maximumSize" >
         <size>
          <width>125</width>
          <height>116</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="../resources/resources.qrc" >:/header/images/InformationLogo.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lblHeader" >
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;title>xVideoServiceThief&lt;/title>&lt;/head>&lt;body>&lt;div align="center">&lt;p>&lt;span style="font-size: 24px; font-family: Arial, Helvetica, sans-serif; font-weight: bold; color: #666666;">xVideoServiceThief&lt;/span>&lt;br />&lt;span style="font-family: Arial, Helvetica, sans-serif;font-size: 13px;">Powered by Xesc &amp;amp; Technology &amp;reg; 2008&lt;/span>&lt;/p>&lt;/div>&lt;/body>&lt;/html></string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="openExternalLinks" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="lblProgramVersionTitle" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Program version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblProgramVersion" >
              <property name="font" >
               <font/>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="lblLanguageTitle" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Language:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblLanguage" >
              <property name="font" >
               <font/>
              </property>
              <property name="text" >
               <string>&lt;b>English (UK)&lt;/b></string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="lblURLTitle" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Homepage:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblURL" >
              <property name="frameShadow" >
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text" >
               <string>&lt;a href="http://xviservicethief.sourceforge.net/">http://xviservicethief.sourceforge.net/&lt;/a></string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblInformation" >
     <property name="text" >
      <string>xVideoServiceThief is a tool for downloading your favourite video clips from a lot of video websites.

xVideoServiceThief also provide you the ability to convert each video in most popular formats: AVI, MPEG1, MPEG2, WMV, MP4, 3GP, MP3 file formats.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="indent" >
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblWebsitesTitle" >
     <property name="text" >
      <string>&lt;b>xVideoServiceThief currently supports download from:&lt;/b></string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QTextBrowser" name="textBrowser" >
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-family:'Arial'; font-size:large;">Total websites supported (05/02/2008): &lt;/span>&lt;span style=" font-family:'Arial'; font-size:large; font-weight:600;">35&lt;/span>&lt;span style=" font-family:'Arial'; font-size:large;"> &lt;/span>&lt;/p>
&lt;ul style="-qt-list-indent: 1;">&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">123video &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">5min &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">aBum &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Aniboom &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Bebo &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Break &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Broadcaster &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Caught On Video &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">CinemaVIP&lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Clip4e &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Dailymotion &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Dumpalink &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Glumbert &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Google Video &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">LiveVideo &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Lulu TV &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Metacafe &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">MySpaceTv&lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Sclipo &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Spike&lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Tu.tv&lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Video.ca &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">VideoLeak &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Yahoo Video (*) &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Yikers &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">YouTube &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">ZappInternet &lt;/span>&lt;/li>&lt;/ul>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial'; font-size:8pt;">&lt;span style=" font-size:large;">Adult web sites: &lt;/span>&lt;/p>
&lt;ul style="-qt-list-indent: 1;">&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Badjojo &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">DaleAlPlay &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">DesiSexTube &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">PornoTube &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Shufuni&lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">TuPorno &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">YouPorn &lt;/span>&lt;/li>
&lt;li style=" font-family:'Arial'; font-size:8pt;" style=" margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:large;">Yuvutu &lt;/span>&lt;/li>&lt;/ul>&lt;/body>&lt;/html></string>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="Line" name="bvlSeparator" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>311</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCredits" >
         <property name="minimumSize" >
          <size>
           <width>106</width>
           <height>25</height>
          </size>
         </property>
         <property name="text" >
          <string>Credits...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddVideo" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>106</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>Close</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>btnAddVideo</sender>
   <signal>clicked()</signal>
   <receiver>Information</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>364</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel" >
     <x>407</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
